.travis.yml

```diff
@@ -18,10 +18,18 @@ matrix:
       env: TARGET=cs_dry_run
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
+    # test 2.7 LTS
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*
+    # test 2.8 LTS
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*
+    # test the latest stable 3.x release
+    - php: 5.6
+      env: SYMFONY_VERSION=^3.0
+    # test the latest release (including beta releases)
+    - php: 7.1
+      env: DEPENDENCIES=beta
```

Comment:
Please use a `DEPENDENCIES=dev` instead of `beta`. It would make the job much more useful.